docs/source/config-file.mdx

```diff
@@ -180,25 +157,25 @@ The available fields depend on the value of the nested `type` key:
 
 ##### stdio (default)
 
-| Option    | Value               | Default Value | Description                                                                                                              |
-| :-------- | :------------------ | :------------ | :----------------------------------------------------------------------------------------------------------------------- |
-| `type`    | `"stdio"`           | \*             | Use standard IO for communication between the server and client                                                          |
+| Option | Value     | Default Value | Description                                                     |
```

Comment:
The tables in this section use slightly different formats, which could be confusing for users.

<img width="1530" height="2146" alt="Image" src="https://github.com/user-attachments/assets/5696edb7-7248-4cc0-afaf-eb075539200c" />

Reply:
I'll log this in a separate ticket, good catch!